winch/codegen/src/isa/x64/masm.rs

```diff
@@ -404,6 +404,10 @@ impl Masm for MacroAssembler {
         }
     }
 
+    fn float_sqrt(&mut self, dst: Reg, src: RegImm, size: OperandSize) {
```

Comment:
I almost certain that this can be simplified to:

```rust 
fn float_sqrt(&mut self, dest: Reg, src: Reg, size: OperandSize { .. }
```
because this instruction can take either two registers or a register and a memory operand https://www.felixcloutier.com/x86/sqrtsd